packages/@okta/vuepress-site/docs/guides/submit-app/main/index.md

```diff
@@ -60,13 +60,48 @@ If the Okta OIN team identifies any issues in the initial review and QA testing
 
   >**Note:** SWA app integrations are no longer accepted for publication in the OIN catalog. However, existing SWA apps are still maintained by the OIN team.
 
+### Understand the submission review process
+
+The submission review process begins after you click **Submit for Review** in the [OIN Manager](https://oinmanager.okta.com). Okta sends you an email notification that your integration is now queued for review by the Okta OIN team, along with a date by which you should complete the initial review.
+
+The OIN Manager always shows the current status of your integration.
+
+#### Step 1: Initial Review
+
+* **Pending review by Okta**: The Okta OIN team is notified of your submission. Okta reviews the submission and notifies you by email when the submission review is complete.
+* **Action required**: Okta has reviewed your submission and found issues that require your attention. Check your email for results from the Okta initial review. Sign in to OIN Manager, update the requested details, and click **Submit for Review**. After the submission review is complete, the Okta OIN team moves it to Step 2 for QA testing.
```

Comment:
Maybe:
"After the submission review is complete..."
>
"After submitting your changes, the Okta OIN team moves your submission to Step 2 for QA testing."
or maybe
"After submitting your changes, the Okta OIN team moves your submission to step 2."

Reply:
I'm updating it to:

> After the OIN team reviews your updated submission and verifies that the issues have been resolved, the submitted integration moves to step 2 for QA testing.

let me know what you think?